scripts/variantstore/wdl/GvsSitesOnlyVCF.wdl

```diff
@@ -35,49 +63,188 @@ workflow GvsSitesOnlyVCF {
             output_annotated_file_name = "${output_annotated_file_name}_${i}",
             nirvana_data_tar = nirvana_data_directory,
-            custom_annotations_file = ExtractAnAcAfFromVCF.annotations_file
+            custom_annotations_file = ExtractAnAcAfFromVCF.annotations_file,
+            subpopulation_calculations_files = ExtractAcAnAfFromSubpopulationVCFs.annotations_file
         }
 
-       call PrepAnnotationJson {
-         input:
-           annotation_json = AnnotateVCF.annotation_json,
-           output_file_suffix = "${i}.json.gz",
-           output_path = output_path,
-           service_account_json_path = service_account_json_path,
-       }
-    }
-
-     call BigQueryLoadJson {
-         input:
-             nirvana_schema = vat_schema_json_file,
-             vt_schema = variant_transcript_schema_json_file,
-             genes_schema = genes_schema_json_file,
-             project_id = project_id,
-             dataset_name = dataset_name,
-             output_path = output_path,
-             table_suffix = table_suffix,
-             service_account_json_path = service_account_json_path,
-             prep_jsons_done = PrepAnnotationJson.done
-         }
-
-     call BigQuerySmokeTest {
-         input:
-             project_id = project_id,
-             dataset_name = dataset_name,
-             counts_variants = ExtractAnAcAfFromVCF.count_variants,
-             table_suffix = table_suffix,
-             service_account_json_path = service_account_json_path,
-             load_jsons_done = BigQueryLoadJson.done
-         }
+        call PrepAnnotationJson {
+          input:
+            annotation_json = AnnotateVCF.annotation_json,
+            output_file_suffix = "${i}.json.gz",
+            output_path = output_path,
+            service_account_json_path = service_account_json_path
+        }
+    }
+
+    call BigQueryLoadJson {
+       input:
+         nirvana_schema = vat_schema_json_file,
+         vt_schema = variant_transcript_schema_json_file,
+         genes_schema = genes_schema_json_file,
+         project_id = project_id,
+         dataset_name = dataset_name,
+         output_path = output_path,
+         table_suffix = table_suffix,
+         service_account_json_path = service_account_json_path,
+         prep_jsons_done = PrepAnnotationJson.done
+  }
+
+    call BigQuerySmokeTest {
+       input:
+         project_id = project_id,
+         dataset_name = dataset_name,
+         counts_variants = ExtractAnAcAfFromVCF.count_variants,
+         table_suffix = table_suffix,
+         service_account_json_path = service_account_json_path,
+         load_jsons_done = BigQueryLoadJson.done
+   }
 }
 
 ################################################################################
+
+task MakeSubpopulationFiles { ## TODO why am I seemingly able to get the ancestry file in Terra? is it cached? I should need the SA for it---so actually, but me worth trying the SA use sooner
```

Comment:
Is this still an issue?

Reply:
Not an issue--but I would feel more comfortable with the data in an AoU bucket just for completeness I suppose?  I'm going to do a follow on pr where I move the ancestry file into an AoU protected bucket and add a SA json to this step